### ISSUE TITLE
Can we make it realtime?

### ISSUE THREAD

**Issue by Bimal2614** [KwaiVGI/LivePortrait#303]
I want to make something like realtime audio will be coming, and there will be one face, Need something like If I say Hi and I have incorporated OpenAI with this, openAI will generate Hello, this image should speak with emotion "Hello"

**Comment by Mayorc1978** [comment]
Yes, I would like to know as well. At least for image editing it should be almost instantaneous. But it's not. So I don't think with videos would be that easy as well.

**Comment by warmshao** [comment]
try this https://github.com/warmshao/FasterLivePortrait

**Comment by cleardusk** [comment]
@Bimal2614 real-time is possible, but it requires lots of engineering efforts. [FasterLivePortrait](https://github.com/warmshao/FasterLivePortrait) is a good start 👍

**Comment by Bimal2614** [comment]
I am looking for a realtime means, I have camera open in my system, and I will do movements and in it will be reflected to that image, is that possible with it?
@cleardusk
@Mayorc1978 
@warmshao 
@jeethu
